docs/ArgoCD Integration.md

```diff
@@ -147,15 +157,17 @@ repoServer:
 
 # Configuration of ArgoCD
 
-When using private key encryption it is required to configure ArgoCD so that it has access to the private key to decrypt the encrypted value file(s). When using GCP KMS, encrypted value file(s) can be decrypted using [Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
+When using private key encryption it is required to configure ArgoCD repo server so that it has access 
+to the private key to decrypt the encrypted value file(s). When using GCP KMS, encrypted value file(s)
+can be decrypted using [Application Default Credentials](https://developers.google.com/identity/protocols/application-default-credentials).
 
 ## Private key encryption
 There are two ways to configure ArgoCD to have access to your private key.
-Either mount the gpg key from secret as volume or access the fetch the private key directly from a kubernetes secret.
-Both methods depend on a secret holding the key in asci format.
+Either mount on the argocd-repo-server the gpg/age key from secret as volume or access the fetch the
+private key directly from a kubernetes secret. Both methods depend on a secret holding the key in ASCII format.
```

Comment:
I would rephrase this sentence like:

> There are two ways to configure ArgoCD to have access to your private key:
> 
> - mount the PGP/age key secret as a volume in the `argocd-repo-server`; or
> - fetch the secret value (private key) directly using Kubernetes API.
> 
> Both methods depend on a Kubernetes secret holding the key in plain-text format (i.e., not encrypted).

I assumed that for "ASCII" you mean "not encrypted". Correct me if I'm wrong :-)

Reply:
I mean the ASCII armored format here - https://unix.stackexchange.com/questions/623375/what-is-the-armored-option-for-in-gnupg.

Reply:
Ah ok, my bad :-D